tests/link/test_numba.py

```diff
@@ -86,6 +115,67 @@ def test_Elemwise(inputs, input_vals, output_fn):
     compare_numba_and_py(out_fg, input_vals)
 
 
+@pytest.mark.parametrize(
+    "unary_op, x_range",
+    [
+        (inv, list(range(1, 10))),
```

Comment:
The tests in `binary.py` check for a much larger interval. But that would make these tests a bit slow. So shortened the interval as we only need to see the functionality of the Numba `Op`.

Reply:
Or you can use `np.linspace` on a large interval, with a specified number of points? Then they also take float values inbetween.